alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/action/Throttle.kt

```diff
@@ -41,6 +41,7 @@ data class Throttle(
     companion object {
         const val VALUE_FIELD = "value"
         const val UNIT_FIELD = "unit"
+        const val MAX_THROTTLE_VALUE = 1440 // 1440 minutes = one day
```

Comment:
Lets make sure this can also work when we introduce new Unit. Can you add validation in the constructor of `Throttle` or before creating the `Throttle` Object so you have both Value and Unit when checking for max throttle value?

Reply:
We only support MINUTES unit currently, already has checking now.

require(StringUtils.equals(unitString, ChronoUnit.MINUTES.name), { "Only support MINUTES throttle unit currently" })